src/rules.js

```diff
@@ -169,8 +169,8 @@ export const inline = {
     lDelim: /^(?:\*+(?:([punct_])|[^\s*]))|^_+(?:([punct*])|([^\s_]))/,
     //        (1) and (2) can only be a Right Delimiter. (3) and (4) can only be Left.  (5) and (6) can be either Left or Right.
     //          () Skip orphan inside strong  () Consume to delim (1) #***                (2) a***#, a***                   (3) #***a, ***a                 (4) ***#              (5) #***#                 (6) a***a
```

Comment:
Could you also update this comment line by repositioning these labels above their respective sections to help future developers parse the regex?

Reply:
Done.